packages/expo-location/src/Location.types.ts

```diff
@@ -358,10 +358,17 @@ export type LocationHeadingObject = {
    */
   magHeading: number;
   /**
-   * Level of calibration of compass.
-   * - `3`: high accuracy, `2`: medium accuracy, `1`: low accuracy, `0`: none
+   * Level of calibration of compass:
+   * - `3`: high accuracy,
+   * - `2`: medium accuracy,
+   * - `1`: low accuracy,
+   * - `0`: none.
```

Comment:
```suggestion
   * - `3`: high accuracy
   * - `2`: medium accuracy
   * - `1`: low accuracy
   * - `0`: none
```

Since these are being listed, let's remove `,` and `.` as it's not going to be a sentence anymore.